osquery/database/tests/database_tests.cpp

```diff
@@ -72,4 +72,13 @@ TEST_F(DatabaseTests, test_delete_values) {
   EXPECT_FALSE(s.ok());
   EXPECT_TRUE(value.empty());
 }
+
+TEST_F(DatabaseTests, test_reset) {
+  setDatabaseValue(kLogs, "reset", "1");
+  resetDatabase();
+
+  std::string value;
+  EXPECT_TRUE(getDatabaseValue(kLogs, "reset", value));
```

Comment:
This expectation only exists for specific database plugins. The `ephemeral` plugin's reset will wipe state.